packages/protocol/test/common/metatransactionwallet.ts

```diff
@@ -591,4 +599,30 @@ contract('MetaTransactionWallet', (accounts: string[]) => {
       })
     })
   })
+
+  describe('#transferCeloDollarsToSigner', () => {
+    const value = 100
+    let mockStableToken: MockStableTokenInstance
+    beforeEach(async () => {
+      const Registry: RegistryContract = artifacts.require('Registry')
+      const registry: RegistryInstance = await Registry.at(
+        '0x000000000000000000000000000000000000ce10'
+      )
+      // Problem is this is owned by governance...
+      console.log(await registry.owner())
+      console.log(accounts)
```

Comment:
Assume these are debugging logs/comments that should be removed from final version?

Reply:
Yeah tests are still WIP

Reply:
Namely, I can't register the mock token b/c the registry is owned by governance